.travis.yml

```diff
@@ -47,6 +47,11 @@ addons:
             - python3-gst-1.0
             - unrar
 
+before_install:
+    - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu xenial main"
+    - sudo apt-get -y update
+    - sudo apt-get install sqlite3 -y
```

Comment:
Hmm… I suppose it's a somewhat troubling indication about the popularity of SQLite JSON support if we need to override the apt repository.